hcl2/version.py

```diff
@@ -1,2 +1,3 @@
-"""Version module"""
-__version__ = '0.3.28'
+"""Place of record for the package version"""
+
+__version__ = "3.0.1"
```

Comment:
@schosterbarak just want to make sure - this change shouldn't break anything on our side right?